client/ayon_core/tools/sceneinventory/model.py

```diff
@@ -321,7 +321,17 @@ def _add_containers(self, containers, parent=None):
             )
             group_node["representation"] = repre_id
             group_node["version"] = version_entity["version"]
-            group_node["highest_version"] = highest_version["version"]
+
+            # Check if the version is outdated. If the version is below 0 it
+            # is a hero version and will never be considered outdated
+            is_outdated = False
+            if version_entity["version"] >= 0:
+                last_version = highest_version_by_product_id.get(
+                    version_entity["productId"])
+                if last_version is not None:
+                    is_outdated = version_entity["version"] != last_version
+            group_node["isOutdated"] = is_outdated
+
```

Comment:
```
            version = version_entity["version"]
            if version < 0:
                version = HeroVersionType(version)
            group_node["version"] = version

            # Check if the version is outdated. If the version is below 0 it
            # is a hero version and will never be considered outdated
            is_outdated = False
            if not isintance(version, HeroVersionType):
                last_version = highest_version_by_product_id.get(
                    version_entity["productId"])
                if last_version is not None:
                    is_outdated = version_entity["version"] != last_version
            group_node["isOutdated"] = is_outdated
```

Reply:
Feel free to change lines and make it "nicer".

Reply:
Thanks - seemed good to me - just changed comments a bit.